Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/DriverOnboarding.hs

```diff
@@ -81,5 +93,11 @@ validateImage (personId, merchantId) = withFlowHandlerAPI . Image.validateImage
 validateImageFile :: (Id DP.Person, Id DM.Merchant) -> Image.ImageValidateFileRequest -> FlowHandler Image.ImageValidateResponse
 validateImageFile (personId, merchantId) = withFlowHandlerAPI . Image.validateImageFile False (personId, merchantId)
 
+generateAadhaarOtp :: (Id DP.Person, Id DM.Merchant) -> AadhaarVerification.AadhaarOtpReq -> FlowHandler AadhaarVerification.AadhaarVerificationResp
+generateAadhaarOtp (personId, merchantId) = withFlowHandlerAPI . AV.generateAadhaarOtp False Nothing (personId, merchantId)
+
+verifyAadhaarOtp :: (Id DP.Person, Id DM.Merchant) -> AV.VerifyOtpReq -> FlowHandler AadhaarVerification.AadhaarOtpVerifyRes
+verifyAadhaarOtp (personId, merchantId) = withFlowHandlerAPI . AV.verifyAadhaarOtp False Nothing (personId, merchantId)
```

Comment:
Why are you passing merchant and merchantId both ?
And always you are hardcoding merchant to Nothing.